Mention libxcb-xfixes0-dev and libxcb-shape0-dev in README

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
Yes. `cargo install kdash` will fail on a fresh install of Ubuntu Desktop, with `xorg-dev` installed as per the README, because of missing dependencies

**Describe the solution you'd like**
A clear and concise description of what you want to happen.
The solution to the issue is to install `libxcb-xfixes0-dev` and `libxcb-shape0-dev`, the non-`-dev` packages are not enough. The proposed solution is to add the following to the README for Ubuntu/Debian:
```
sudo apt install -y libxcb-xfixes0-dev and libxcb-shape0-dev
```

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.
N.A.

**Additional context**
Add any other context or screenshots about the feature request here.

Output from rustc:
```
   Compiling kdash v0.2.4
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "/tmp/cargo-installGcVkvt/release/deps/kdash-3cd074a07c9a7e61.kdash.d81f41e3-cgu.0.rcgu.o" "-Wl,--as-needed" "-L" "/tmp/cargo-installGcVkvt/release/deps" "-L" "/tmp/cargo-installGcVkvt/release/build/openssl-sys-8b6d1c8eda31ec5e/out/openssl-build/install/lib" "-L" "/tmp/cargo-installGcVkvt/release/build/ring-7cd5715780f3a214/out" "-L" "/home/tobias/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/tmp/rustciKrNKZ/libopenssl_sys-91ac39aed5aff684.rlib" "/tmp/rustciKrNKZ/libring-e3238ae251d7e213.rlib" "-Wl,--start-group" "-Wl,--end-group" "/home/tobias/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-dd7db1bec6909f24.rlib" "-Wl,-Bdynamic" "-lxcb" "-lxcb-render" "-lxcb-shape" "-lxcb-xfixes" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" "/home/tobias/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-o" "/tmp/cargo-installGcVkvt/release/deps/kdash-3cd074a07c9a7e61" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs"
  = note: /usr/bin/ld: cannot find -lxcb-shape
          /usr/bin/ld: cannot find -lxcb-xfixes
          collect2: error: ld returned 1 exit status
```

--- a/README.md
+++ b/README.md
@@ -108,7 +108,7 @@ cargo install kdash
 
 You can also clone the repo and run `cargo run` or `make` to build and run the app
 
-**Note:**: On Linux you might need to have package `xorg-dev` (Debian/Ubuntu) or `xorg-x11-server-devel` (Fedora) or equivalent installed for the copy to clipboard features to work
+>Note: On Linux you might need to have package `xorg-dev` (Debian/Ubuntu) or `xorg-x11-server-devel` (Fedora) or equivalent installed for the copy to clipboard features to work
 
 ## USAGE:
 

